docs/getting-started/quickstart.md

```diff
@@ -46,24 +46,20 @@ Much of Prefect's functionality is backed by an API - if [self-hosting](/guides/
 
 
 
-### Step 3: Create a flow
-**The fastest way to get started with Prefect is to add a `@flow` decorator to any python function**.
+### Step 3: Write a flow
+The fastest way to get started with Prefect is to add a `@flow` decorator to any Python function and call its `serve` method to create a deployment.  
```

Comment:
```suggestion
The fastest way to get started with Prefect is to add a `@flow` decorator to any Python function to turn it into a flow, an observable, deployable piece of work that can be run remotely via a schedule or a remote based trigger.  
```

Reply:
I think we should make some attempt to explain what a flow is.

Reply:
Makes sense, let me think about what to say here